packages/renderer-commonmark/test/commonmark-test.ts

```diff
@@ -114,10 +117,7 @@ After all the lists`);
       });
 
       let renderer = new CommonMarkRenderer();
-      expect(renderer.render(document)).toBe(
-                   '> This is a quote\n> ' + `
-> That has some
-> lines in it.`);
+      expect(renderer.render(document)).toBe('> This is a quote\n> \n> That has some\n> lines in it.\n\n');
```

Comment:
the whitespace changes make it slightly hard to compare during review but I think it's just whitespace and added trailing newlines?